tsdb/engine/b1/b1.go

```diff
@@ -549,32 +549,37 @@ type Tx struct {
 	engine *Engine
 }
 
-// Cursor returns an iterator for a key.
-func (tx *Tx) Cursor(key string, direction tsdb.Direction) tsdb.Cursor {
-	// Retrieve key bucket.
-	b := tx.Bucket([]byte(key))
+// Cursor returns an iterator for a key over a single field.
```

Comment:
I may be reading this wrong, but we say `single field` here, do we mean `single point`?  Later, in the `cursor.read` method we check to see if we have a single field or multiple fields, which implies we can cursor over multiple fields right?